<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>609</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>765</width>
    <height>609</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>765</width>
    <height>609</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>743</width>
        <height>499</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>743</width>
        <height>499</height>
       </size>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(85, 170, 255);
background-color: rgb(0, 170, 255);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <widget class="QWidget" name="nuevoPaciente">
       <attribute name="icon">
        <iconset>
         <normaloff>nuevo.png</normaloff>nuevo.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Nuevo Paciente</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>100</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Nombre:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>140</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Cedula:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>190</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Edad:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>230</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Patolog√≠a:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="campoNombre">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>101</y>
          <width>301</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="campoCC">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>150</y>
          <width>301</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="campoEdad">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>200</y>
          <width>301</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="campoPatologia">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>240</y>
          <width>301</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="botonIngresar">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>330</y>
          <width>121</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Enviar</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>nuevo.png</normaloff>nuevo.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>330</y>
          <width>141</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Cancelar</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>limpiar.png</normaloff>limpiar.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>100</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Nombre:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="icon">
        <iconset>
         <normaloff>6861362.png</normaloff>6861362.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Borrar un paciente</string>
       </attribute>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>170</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Cedula:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="campoCCborrar">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>180</y>
          <width>301</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>70</y>
          <width>501</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Parte de borrado de Pacientes&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;por favor brinde la cedula del paciente&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="botonBorrar">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>320</y>
          <width>121</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Borrar</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>6861362.png</normaloff>6861362.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_13">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>320</y>
          <width>111</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Cancelar</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>limpiar.png</normaloff>limpiar.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="icon">
        <iconset>
         <normaloff>lupa1.png</normaloff>lupa1.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Ver Paciente</string>
       </attribute>
       <widget class="QLineEdit" name="nombreBuscar">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>340</y>
          <width>451</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>330</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Nombre:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tablaPacientes">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>591</width>
          <height>291</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>Nombre</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Edad</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Nueva columna</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Patologia</string>
         </property>
        </column>
       </widget>
       <widget class="QPushButton" name="botonBuscar">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>380</y>
          <width>141</width>
          <height>51</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>18</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Buscar</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>lupa1.png</normaloff>lupa1.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="clickver">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>110</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>click para ver</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="actualizarPacinte">
       <attribute name="icon">
        <iconset>
         <normaloff>refex.png</normaloff>refex.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Actualizar Informaci√≥n</string>
       </attribute>
       <widget class="QWidget" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>50</y>
          <width>611</width>
          <height>271</height>
         </rect>
        </property>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>150</y>
           <width>121</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Patolog√≠a:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>110</y>
           <width>121</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Edad:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="edadActualizar">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>120</y>
           <width>301</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>20</y>
           <width>121</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Nombre:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="ccActualizar">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>70</y>
           <width>301</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="patologiaActualizar">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>160</y>
           <width>301</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="nombreActualizar">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>21</y>
           <width>301</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>60</y>
           <width>121</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Cedula:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_8">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>210</y>
           <width>121</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Cancelar</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>limpiar.png</normaloff>limpiar.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="enviarActualizar1">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>210</y>
           <width>121</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Enviar</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>nuevo.png</normaloff>nuevo.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="botonSalirSis">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Salir del sistema</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>765</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>campoCC</tabstop>
  <tabstop>campoEdad</tabstop>
  <tabstop>campoPatologia</tabstop>
  <tabstop>botonIngresar</tabstop>
  <tabstop>pushButton_7</tabstop>
  <tabstop>campoNombre</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>campoPatologia</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>campoEdad</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>468</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_13</sender>
   <signal>clicked()</signal>
   <receiver>campoCCborrar</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>456</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>patologiaActualizar</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>edadActualizar</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>ccActualizar</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>nombreActualizar</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>campoCC</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>campoNombre</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
